Source/QED/BreitWheelerDummyTable.H

```diff
@@ -0,0 +1,78 @@
+#ifndef WARPX_breit_wheeler_dummy_tables_h_
+#define WARPX_breit_wheeler_dummy_tables_h_
+
+#include "BreitWheelerEngineInnards.H"
+
+#include <AMReX_REAL.H>
+
+#include <limits>
+#include <vector>
+
+namespace QedUtils{
+
+//A default mini-table used for test purposes
+const struct //BreitWheelerEngineInnardsDummy
+{
+    picsar::multi_physics::breit_wheeler_engine_ctrl<amrex::Real> ctrl{
+        0.001, //chi_phot_min
+        0.1, //chi_phot_tdndt_min
+        200, //chi_phot_tdndt_max
+        64, //chi_phot_tdndt_how_many
+        0.01, //chi_phot_tpair_min
+        200, //chi_phot_tpair_max
+        2, //chi_phot_tpair_how_many
+        2, //chi_frac_tpair_how_many
+    };
```

Comment:
In some conditions, I am wondering if this could lead to some bugs. I think @oshapoval had issues with a function call pr definition using
```c++
f(
  a, // parameter a
  b);
```
that was fixed by removing the comment.

An alternative is below, which I think is more stable?
```c++
    picsar::multi_physics::breit_wheeler_engine_ctrl<amrex::Real> ctrl{
         0.001, /*chi_phot_min*/
         0.1, /*chi_phot_tdndt_min*/
         200, /*chi_phot_tdndt_max*/
         64, /*chi_phot_tdndt_how_many*/
         0.01, /*chi_phot_tpair_min*/
         200, /*chi_phot_tpair_max*/
         2, /*chi_phot_tpair_how_many*/
         2, /*chi_frac_tpair_how_many*/
     };
```

Reply:
I've just implemented your solution. I agree that it could be safer.